components/common/useVisible.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
```

Comment:
nit: license

Reply:
I might have a play with eslint to see if I can make it check this.